workspace/versions.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0"
```

```suggestion
  required_version = ">= 1.7"
```
Let's bump this to match [the min reqs we advertise](https://github.com/opslevel/terraform-provider-opslevel/?tab=readme-ov-file#minimum-requirements)